reversestring-GEZT/prompts.md

```
@@ -0,0 +1,8 @@
+# Prompts que construyen la página web Inversor de texto
+***
+## El chatbot que se uso fue Claude ai de Anthropic y el LLM utilizado fue Sonnet 4.
+---
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: ***; Actual: ---

(MD035, hr-style)